README.md

```diff
@@ -169,82 +188,90 @@ In the following section we guide you trough installing and running a Gazebo sim
 
    # Build and run simulation
    make px4_sitl_default gazebo
+   
+   # Quit the simulation (Ctrl+C)
 
-   # Setup some more Gazebo-related environment variables (You may need to modify this line based on the location of the Firmware folder on your machine)
+   # Setup some more Gazebo-related environment variables (modify this line based on the location of the Firmware folder on your machine)
    . ~/Firmware/Tools/setup_gazebo.bash ~/Firmware ~/Firmware/build/px4_sitl_default
    ```
 
-1. Add the Firmware directory to ROS_PACKAGE_PATH so that ROS can start PX4.
-
+1. Add the Firmware directory to ROS_PACKAGE_PATH so that ROS can start PX4:
    ```bash
    export ROS_PACKAGE_PATH=${ROS_PACKAGE_PATH}:~/Firmware
    ```
 1. Finally, set the GAZEBO_MODEL_PATH in your bashrc:
-```bash
-echo export GAZEBO_MODEL_PATH=${GAZEBO_MODEL_PATH}:~/catkin_ws/src/avoidance/avoidance/sim/models:~/catkin_ws/src/avoidance/avoidance/sim/worlds >> ~/.bashrc
-```
+   ```bash
+   echo export GAZEBO_MODEL_PATH=${GAZEBO_MODEL_PATH}:~/catkin_ws/src/avoidance/avoidance/sim/models:~/catkin_ws/src/avoidance/avoidance/sim/worlds >> ~/.bashrc
+   ```
 
-Steps 3, 4 and 5 together with sourcing your catkin setup.bash (`source ~/catkin_ws/devel/setup.bash`) should be repeated each time a new terminal window is open.
+The last three steps, together with sourcing your catkin **setup.bash** (`source ~/catkin_ws/devel/setup.bash`) should be repeated each time a new terminal window is open.
 You should now be ready to run the simulation using local or global planner.
 
 ### Local Planner (default, heavily flight tested)
 
 This section shows how to start the *local_planner* and use it for avoidance in mission or offboard mode.
 
-The planner is based on the [3DVFH+](http://ceur-ws.org/Vol-1319/morse14_paper_08.pdf) algorithm. To run the algorithm it is possible to
+The planner is based on the [3DVFH+](http://ceur-ws.org/Vol-1319/morse14_paper_08.pdf) algorithm.
 
-* simulate a forward looking stereo camera running OpenCV's block matching algorithm (SGBM by default)
-
-   ```bash
-   # if stereo-image-proc not yet installed
-   sudo apt install ros-kinetic-stereo-image-proc
-
-   roslaunch local_planner local_planner_stereo.launch
-   ```
-
-   The disparity map from `stereo-image-proc` is published as a [stereo_msgs/DisparityImage](http://docs.ros.org/api/stereo_msgs/html/msg/DisparityImage.html) message, which is not supported by rviz or rqt. To visualize the    message, either run:
-
-   ```bash
-   # if image_view is not yet installed
-   sudo apt install ros-kinetic-image-view
-
-   rosrun image_view stereo_view stereo:=/stereo image:=image_rect_color
-   ```
+You *may* need to install some additional dependencies (if not already installed):
```

Comment:
FYI pulled these to beginning. For me they were already installed. But having them here makes the rest flow better.